progs/incorrect/set_multiple.sched

```diff
@@ -0,0 +1,5 @@
+classes A, B;
+
+combined = fifo[A, B];
```

Comment:
I see, so this is incorrect because we instead need `fifo[union[A, B]]`. Would you mind noting this fact in a comment in this file?

Reply:
Sure!